libraries/botbuilder/tests/streaming/botFrameworkAdapterStreaming.test.js

```diff
@@ -44,12 +51,52 @@ describe('BotFrameworkAdapter Streaming tests', () => {
     it('starts and stops a namedpipe server', () => {
         const adapter = new BotFrameworkAdapter();
 
-        adapter.useNamedPipe('PipeyMcPipeface', async (context) => {
-            await bot.run(context);
+        adapter.useNamedPipe('PipeyMcPipeface', async (_) => {
+            // no op
         });
         expect(adapter.streamingServer.disconnect()).to.not.throw;
```

Comment:
Does this need an `await` somewhere? Or is `disconnect()` sync.

Reply:
Good question! `disconnect()` is sync, so this works